nebula-logger/core/main/configuration/classes/LoggerCache.cls

```diff
@@ -182,8 +188,11 @@ public without sharing class LoggerCache {
         public void put(String key, Object value) {
             this.transactionCache.put(key, value);
 
-            // Platform cache does not support storing null values
```

Comment:
maybe it's just me, but I think I prefer this approach over the committed change here. `null` as a case to be handled is a fairly common thing, whereas by using a sentinel value and hot-swapping it in the `get`, we're adding overhead to _every_ cached value we attempt to access.

Reply:
Just to follow up after we chatted - I'm going to stick with the current approach of using the sentinel value since there's some goofiness with platform cache, the `LoggerCache` internally still prefers using the transaction cache (even when platform cache is enabled & available). Thanks again for talking through this!